src/Components/PreviewSignature/PreviewSignature.vue

```diff
@@ -26,7 +26,6 @@ export default {
 		canvas: null,
 		isDrawing: false,
 		color: '#000000',
-		src: null,
```

Comment:
this define src and no allow use in template?

Reply:
src already was defined at `props`, the reported error is that isn't ok to use the same property name as `data`. I only removed from data and worked to me.